lib/vaahextendflutter/services/storage/local/extendable.dart

```diff
@@ -0,0 +1,114 @@
+import '../storage_response.dart';
+import 'storage.dart';
+
+/// [LocalStorageExtendable] can be extended to have more class features other than the available
+/// features.
+///
+/// Example:
+///
+/// ```dart
+/// class ELS extends LocalStorageExtendable {
+///   void myFunction(){
+///     // logic
+///   }
+/// }
+///
+/// final ELS els = ELS();
+///
+/// void useCase(){
+///   final response = await els.create(
+///     collectionName: 'posts',
+///     key: 'key',
+///     value: 'value',
+///   ); // existing method
+///
+///   final newResponse = await els.myFuction(); // added method
+/// }
+///
+/// ```
+class LocalStorageExtendable {
+  static const String defaultCollectionName = 'vaah-fluter-box';
```

Comment:
no need to create extra class, you still can access your custom function and other crud functions by doing this 
```dart
class CustomStorageService extends LocalStorageWithHive {
  Future<StorageResponse> customFunction(String collectionName, String key) async {
    return read(collectionName: collectionName, key: key);
  }
}

void main() {
  CustomStorageService custom = CustomStorageService();
  custom.deleteAll(collectionName: '');
  custom.customFunction('collection_name', 'key');
}
```